simplified-viewer-preview/src/main/java/org/nypl/simplified/viewer/preview/BookPreviewEmbeddedFragment.kt

```diff
@@ -97,14 +102,40 @@ class BookPreviewEmbeddedFragment : Fragment() {
   }
 
   private fun pauseWebViewPlayer() {
-    webView.loadUrl(
-      "javascript:(function(){" +
-        "button=document.getElementsByClassName('playback-toggle halo')[0];" +
-        "event=document.createEvent('HTMLEvents');" +
-        "event.initEvent('click',true,true);" +
-        "button.dispatchEvent(event);" +
-        "})()"
-    )
+    webView.evaluateJavascript("pausePlayer()", null)
+  }
+
+  private inner class CustomWebViewClient : WebViewClient() {
+    override fun onPageFinished(view: WebView?, url: String?) {
+      injectJavaScript()
+      super.onPageFinished(view, url)
+    }
+
+    private fun injectJavaScript() {
+      try {
+        val inputStream = requireActivity().assets.open("preview_player_commands.js")
+        val buffer = ByteArray(inputStream.available())
+        inputStream.read(buffer)
+        inputStream.close()
+
+        val encodedContent = URLEncoder.encode(String(buffer, Charset.forName(UTF_8)), UTF_8)
+          .replace("+", "%20")
+        val encodedString = Base64.encodeToString(encodedContent.toByteArray(), Base64.NO_WRAP)
+
+        webView.evaluateJavascript(
+          "javascript:(function() {" +
+            "const parent = document.getElementsByTagName('head').item(0);" +
```

Comment:
This can just be `document.head`. The head is special in that `document.head` will always exist, even if there is no explicit `<head>` element in the HTML.